chapters/annotations.tex

```diff
@@ -1389,7 +1389,7 @@ \section{Annotations for the Graphical User Interface}\label{annotations-for-the
 
 If \lstinline!colorSelector = true!, it indicates that an rgb-value selector can be presented for a vector of three elements and generate values 0..255 (the color selector should be useable both for vectors of \lstinline!Integer! and \lstinline!Real!).
 
-Modifying the \fmtannotationindex{Selector} of \lstinline!loadSelector! or \lstinline!saveSelector! displays a file dialog to select a file.
+The presence of \lstinline!loadSelector! or \lstinline!saveSelector! specifying \fmtannotationindex{Selector} provides a file dialog to select a file.
```

Comment:
Doesn't _provides_ have the same issue as _displays_?  Borrowing the formulation from `colorSelector`
```suggestion
The presence of a \fmtannotationindex{Selector} for \lstinline!loadSelector! or \lstinline!saveSelector! indicates that a file dialog can be used to select a file.
```

Alternatively, we could reformulate both `colorSelector` and this to use a phrase based on suggesting something instead:
```suggestion
The presence of a \fmtannotationindex{Selector} for \lstinline!loadSelector! or \lstinline!saveSelector! suggests the use of a file dialog to select a file.
```

Along with:
```
Setting \lstinline!colorSelector = true! suggests the use of an rgb-value selector to pick a vector of three elements with values in the range 0..255 (the color selector should be useable both for vectors of \lstinline!Integer! and \lstinline!Real!).
```

Reply:
This discussion has been continued in response to a later review comment.  Resolving.